packages/@o3r/eslint-plugin/src/index.ts

```diff
@@ -39,19 +39,34 @@ module.exports = {
       }
     },
 
+    // deprecated: should use `angular-template-recommended` instead
     'template-recommended': {
       rules: {
         '@o3r/no-folder-import-for-module': 'error',
         '@o3r/template-async-number-limitation': 'warn'
       }
     },
 
+    // deprecated: should use `monorepo-recommended` instead
     'json-recommended': {
       rules: {
         '@o3r/json-dependency-versions-harmonize': 'error'
       }
     },
 
+    'angular-template-recommended': {
+      rules: {
+        '@o3r/no-folder-import-for-module': 'error',
```

Comment:
This tule as nothing to do in `template` rules

Reply:
It is then wrongly part of `template-recommended`, I will remove it